nuxeo-features/nuxeo-platform-ui-select2/src/main/java/org/nuxeo/ecm/platform/ui/select2/Select2ActionsBean.java

```diff
@@ -108,7 +109,7 @@
         List<String> result = new ArrayList<String>();
         if (array != null) {
             for (int i = 0; i < array.size(); i++) {
-                result.add(array.getJSONObject(i).getString(Select2Common.LABEL));
+                result.add(array.getJSONObject(i).getString(SuggestConstants.LABEL));
```

Comment:
Not sure what `Select2Common` —> `SuggestConstants` has to do with the rest of the changes.